pkg/entities/nfts.go

```diff
@@ -47,18 +47,20 @@ var (
 func (e *Entity) GetNFTDetail(symbol, tokenID string) (*response.IndexerNFTToken, error) {
 	// get collection
 	collection, err := e.repo.NFTCollection.GetBySymbol(symbol)
+	// cannot find collection in db
```

Comment:
handle error not found nft collection in database
can ref https://gorm.io/docs/error_handling.html